includes/class-coblocks-block-assets.php

```diff
@@ -655,6 +655,24 @@ protected function is_page_gutenberg() {
 		}
 		return false;
 	}
+
+	/**
+	 * Determine if the page content contains an element with a coblocks-animate class.
+	 *
+	 * @return boolean True when an element on the page has .coblocks-animate class, else false.
+	 */
+	public function has_coblocks_animation() {
+		global $post;
+
+		/**
+		 * Resolves a fatal error bug on PHP 8+ with Timber.
+		 *
+		 * @see https://wordpress.org/support/topic/the-method-has_masonry_v1_block-produces-a-fatal-error-on-php-8-0-22-and-above/
+		 */
+		$post_content = ! empty( $post ) ? $post->post_content : get_the_content();
```

Comment:
We need to make sure `the_content` filters are applied so that we are working with the output of any server-rendered blocks that have the `coblocks-animate` class applied.

```
$post_content = apply_filters( 'the_content', $post_content );
```